MapboxDirections/MBRouteOptions.swift

```diff
@@ -39,6 +39,10 @@ open class RouteOptions: DirectionOptions {
         super.init(waypoints: waypoints, profileIdentifier: profileIdentifier)
         self.allowsUTurnAtWaypoint = ![MBDirectionsProfileIdentifier.automobile.rawValue, MBDirectionsProfileIdentifier.automobileAvoidingTraffic.rawValue].contains(self.profileIdentifier.rawValue)
     }
+    
+    @objc internal convenience init(matchOptions: MatchingOptions) {
+        self.init(waypoints: matchOptions.waypoints, profileIdentifier: matchOptions.profileIdentifier)
```

Comment:
Do any of the other properties of MatchingOptions need to be carried over? That can happen here in this initializer, after the call to `init(waypoints:profileIdentifier:)`.